.github/workflows/openms_ci_matrix_full.yml

```diff
@@ -138,8 +152,8 @@ jobs:
         # NOTE: x64 is hardcoded. No support for 32bit
         arch: x64
 
-    - if: startsWith(matrix.os, 'macos') || startsWith(matrix.os, 'windows')
-      name: Install Qt (Windows and macOS)
+    - if: startsWith(matrix.os, 'windows')
```

Comment:
Why are we moving away from this action for Mac? I'm unclear how this relates to adding apple silicon.

Reply:
see commit message. qt5 not available.